checks.d/mongo.py

```diff
@@ -6,124 +6,149 @@
+        "metrics.repl.apply.batches.totalMillis": RATE,
+        "metrics.repl.apply.ops": RATE,
+        "metrics.repl.buffer.count": GAUGE,
+        "metrics.repl.buffer.maxSizeBytes": GAUGE,
+        "metrics.repl.buffer.sizeBytes": GAUGE,
+        "metrics.repl.network.bytes": RATE,
+        "metrics.repl.network.getmores.num": RATE,
+        "metrics.repl.network.getmores.totalMillis": RATE,
+        "metrics.repl.network.ops": RATE,
+        "metrics.repl.network.readersCreated": RATE,
+        "metrics.repl.oplog.insert.num": RATE,
+        "metrics.repl.oplog.insert.totalMillis": RATE,
+        "metrics.repl.oplog.insertBytes": RATE,
+        "metrics.repl.preload.indexes.num": RATE,
+        "metrics.repl.preload.indexes.totalMillis": RATE,
+        "metrics.ttl.deletedDocuments": RATE,
+        "metrics.ttl.passes": RATE,
+        "opcounters.command": RATE,
+        "opcounters.delete": RATE,
+        "opcounters.getmore": RATE,
+        "opcounters.insert": RATE,
+        "opcounters.query": RATE,
+        "opcounters.update": RATE,
+        "opcountersRepl.command": RATE,
+        "opcountersRepl.delete": RATE,
+        "opcountersRepl.getmore": RATE,
+        "opcountersRepl.insert": RATE,
+        "opcountersRepl.query": RATE,
+        "opcountersRepl.update": RATE,
+        "replSet.health": GAUGE,
+        "replSet.replicationLag": GAUGE,
+        "replSet.state": GAUGE,
+        "stats.avgObjSize": GAUGE,
+        "stats.collections": GAUGE,
+        "stats.dataSize": GAUGE,
+        "stats.fileSize": GAUGE,
+        "stats.indexes": GAUGE,
+        "stats.indexSize": GAUGE,
+        "stats.nsSizeMB": GAUGE,
+        "stats.numExtents": GAUGE,
+        "stats.objects": GAUGE,
+        "stats.storageSize": GAUGE,
+        "uptime": GAUGE,
+    }
+
+    V2_ONLY_METRICS = {
+        "globalLock.lockTime": GAUGE,
+        "globalLock.ratio": GAUGE,
+        "indexCounters.accesses": RATE,
+        "indexCounters.btree.accesses": RATE,
+        "indexCounters.btree.hits": RATE,
+        "indexCounters.btree.misses": RATE,
+        "indexCounters.btree.missRatio": GAUGE,
+        "indexCounters.hits": RATE,
+        "indexCounters.misses": RATE,
+        "indexCounters.missRatio": GAUGE,
+        "indexCounters.resets": RATE,
+    }
+
+    TCMALLOC_METRICS = {
+        "tcmalloc.generic.current_allocated_bytes": GAUGE,
+        "tcmalloc.generic.heap_size": GAUGE,
+        "tcmalloc.tcmalloc.aggressive_memory_decommit": GAUGE,
+        "tcmalloc.tcmalloc.central_cache_free_bytes": GAUGE,
+        "tcmalloc.tcmalloc.current_total_thread_cache_bytes": GAUGE,
+        "tcmalloc.tcmalloc.max_total_thread_cache_bytes": GAUGE,
+        "tcmalloc.tcmalloc.pageheap_free_bytes": GAUGE,
+        "tcmalloc.tcmalloc.pageheap_unmapped_bytes": GAUGE,
+        "tcmalloc.tcmalloc.thread_cache_free_bytes": GAUGE,
+        "tcmalloc.tcmalloc.transfer_cache_free_bytes": GAUGE,
+    }
+
+    WIREDTIGER_METRICS = {
```

Comment:
The metric names were created by MongoDB. We're simply collecting the metrics that they export. I think it'd be really confusing to remap them to new names because then for every metric we have to go back and figure out what the MongoDB name was to really understand what it's doing.